frontend/src/lib/components/HTMLElementsDisplay/htmlElementsDisplayLogic.ts

```diff
@@ -20,7 +20,7 @@ export interface HtmlElementDisplayLogicProps {
 
 export const elementsChain = (providedElements: ElementType[] | undefined): ElementType[] => {
     const safeElements = [...(providedElements || [])]
-    return safeElements.reverse().slice(Math.max(safeElements.length - 10, 1))
+    return safeElements.reverse()
```

Comment:
Do we want to continue to remove the `body` tag as was done before? Or is it alright to display it?